docs/installation/update.rst

```diff
@@ -2,42 +2,23 @@
 Upgrade to |version|
 ********************
 
-**Last Updated:** June 2017
+**Last Updated:** December 2018
 
-.. warning::
+This document provides a recommendation for how to upgrade Tethys Platform from the last release version. If you have not updated Tethys Platform to the last release version previously, please revisit the documentation for that version and follow those upgrade instructions first.
 
-    UNDER CONSTRUCTION: Pardon our dust, this documentation has not been updated yet. These instructions will NOT work. We apologize for the inconvenience.
 
-1. Get the Latest Version
-=========================
-
-When you installed Tethys Platform you did so using it's remote Git repository on GitHub. To get the latest version of Tethys Platform, you will need to pull the latest changes from this repository:
-
-::
-
-    $ cd /usr/lib/tethys/src
-    $ git pull origin master
-
-2. Install Requirements and Run Setup Script
-============================================
-
-Install new dependencies and upgrade old ones:
+1. Activate Tethys environment and start your Tethys Database:
 
 ::
 
-             $ . /usr/lib/tethys/bin/activate
-    (tethys) $ pip install --upgrade -r /usr/lib/tethys/src/requirements.txt
-    (tethys) $ python /usr/lib/tethys/src/setup.py develop
+    . t
+    tstartdb
```

Comment:
```suggestion
    tstartdb  # If you are using the locally installed database server
    # or
    tethys docker start -c postgis  # If you are using the docker container for your database server
```

Reply:
@swainn Let me know what you think about this update. I'm fine leaving it the way it is, but this may help to highlight that the `tstartdb` command won't start up everyone's database server.